vector/src/main/java/im/vector/app/features/settings/VectorPreferences.kt

```diff
@@ -211,6 +211,9 @@ class VectorPreferences @Inject constructor(
         const val SETTINGS_LABS_ENABLE_THREAD_MESSAGES = "SETTINGS_LABS_ENABLE_THREAD_MESSAGES_FINAL"
         const val SETTINGS_THREAD_MESSAGES_SYNCED = "SETTINGS_THREAD_MESSAGES_SYNCED"
 
+        // This key will be used to enable user for using live user info or not.
+        const val SETTINGS_LABS_USE_LIVE_SENDER_INFO = "SETTINGS_LABS_IS_LIVE_SENDER_INFO"
```

Comment:
Maybe also rename the value

```suggestion
        const val SETTINGS_LABS_USE_LIVE_SENDER_INFO = "SETTINGS_LABS_USE_LIVE_SENDER_INFO"
```